enzy_htp/analysis/spi.py

```diff
@@ -47,7 +47,7 @@ def spi_metric(
         stru = stru.from_single_stru( stru )
 
     result = list()
-    for ss in stru.structures:
+    for ss in stru.structures():
```

Comment:
I'm not crazy about this change. Basically, the error originates from the cases when `get_itself` is the coord_parser given to the `StructureEnsemble` at construction. As such, I think it makes more sense to alter the `structures()` property/function in the class definition. .

Reply:
I take back what I said. At some point `structures()` was changed to not be a `property` and this attribute was not updated. Great catch!